theme/jade/index-partials/header.jade

```diff
@@ -1,9 +1,9 @@
 header.main-header
   img.main-header__logo(src='images/jsunconflogo.svg', alt='JSUnconf')
 
-  p.main-header__title
+  h3.main-header__title
     | Hamburg, 23rd &amp; 24th of April 2016
-  p.main-header__tagline
+  h2.main-header__tagline
```

Comment:
I don't think it's semantically correct to use H-Tags for dates and taglines. These are not really (sub)headings that have a subsection with they describe. These are more like informations on the heading.

Also, we do not have a H1. Why do we have H2/H3?